site/content/help/quickstart/index.md

```diff
@@ -11,11 +11,13 @@ description: A guide designed to get you started with yuzu quickly.
 
 |   	|  CPU 	|  GPU	|  RAM	|
 |-------|-------|-------|-------|
-|**Minimum (for 2D games)**|Intel Core i3-6100 / AMD Ryzen 3 1200|Intel HD Graphics 520 / NVIDIA GeForce GT 1030 / AMD Radeon R7 240|8GB|
-|**Recommended (for 3D games)**|Intel Core i5-8600K / AMD Ryzen 5 3600|NVIDIA GeForce GTX 1060 6GB / AMD Radeon RX 470 8GB|16GB| 
+|**Minimum**|Intel Core i5-4570 / AMD Ryzen 3 1200|Intel HD Graphics 520 / NVIDIA GeForce GT 1030 / AMD Radeon R7 240|8GB|
+|**Recommended**|Intel Core i5-10400 / AMD Ryzen 5 3600|NVIDIA GeForce GTX 1060 6GB / AMD Radeon RX 470 8GB|16GB| 
```

Comment:
```suggestion
|**Recommended**|Intel Core i7-8700 / AMD Ryzen 5 3600|NVIDIA GeForce GTX 1060 6GB / AMD Radeon RX 470 8GB|16GB| 
```

It would be better if we did not suggest this particular CPU as the recommended spec

Reply:
the 8700 is more expensive for the same result, basically, Intel just now has a decent 6 core lineup.